src/gl-state-egl.cpp

```diff
@@ -28,16 +28,31 @@
 #include <iomanip>
 #include <sstream>
 #include <cstring>
-
-#include <EGL/eglext.h>
-
-#ifndef EGL_EXT_platform_base
-typedef EGLDisplay (EGLAPIENTRYP PFNEGLGETPLATFORMDISPLAYEXTPROC) (EGLenum platform, void *native_display, const EGLint *attrib_list);
-#endif
+#include <dlfcn.h>
 
 using std::vector;
 using std::string;
 
+void *get_egl_library()
+{
+    // Use nodelete so the symbols remain after closing.  Useful for tools that lookup symbols
+    // after program terminates, such as some valgrind tools.
+    return dlopen("libEGL.so", RTLD_NOW | RTLD_NODELETE);
```

Comment:
We have to use the SONAME filename form "libEGL.so.1", since the "libEGL.so" file is typically part of development packages and not guaranteed to be installed.

Reply:
Done as described above.